app/helpers/batch_connect/session_contexts_helper.rb

```diff
@@ -3,30 +3,20 @@ def create_widget(form, attrib, format: nil)
     return "" if attrib.fixed?
 
     widget = attrib.widget
-
```

Comment:
I like that you are allowing the form methods to handle all of their own arguments.